.gitignore

```diff
@@ -1,7 +1,7 @@
 # bot (project-specific)
 log/*
 data/*
-_latex_cache/*
+bot/exts/fun/latex/cache/*
```

Comment:
Changing this will require us to alter the deployment manifest too, as by default sir-lance has a read-only file system. We have specifically given a writable volume to _latex_cache/

Reply:
This is more of a note for my reference. I think if we want to change to this new path, we should have it be _cache though, so that the walk_extensions func doesn't even attempt to import it as a module

Reply:
Updated deployment here https://github.com/python-discord/kubernetes/pull/118